controller/api/models.py

```diff
@@ -643,7 +643,8 @@ def create(self):
         kwargs = {'memory': self.release.config.memory,
                   'cpu': self.release.config.cpu,
                   'tags': self.release.config.tags,
-                  'values': self.release.config.values}
+                  'values': self.release.config.values,
+                  'aname': self.app.id}
```

Comment:
I used aname because they were already passing it through in a few other places as aname and wanted to match that. I can change it in all locations though for ease of understanding.